spec/docker_commander_spec.rb

```diff
@@ -62,4 +62,15 @@
     end
   end
 
+  describe '#docker_exec' do
+    it 'executes commands inside docker if container name is specified' do
+      expect(Fastlane::Actions).to receive(:sh).with("docker exec -i my_container bash -l -c \"do stuff\"")
+      @docker_commander.docker_exec(command: 'do stuff', container_name: 'my_container')
+    end
+
+    it 'does not execute the command if the container name is not specified' do
+      expect(Fastlane::Actions).not_to receive(:sh)
+      @docker_commander.docker_exec(command: 'do stuff', container_name: nil)
+    end
+  end
```

Comment:
I know it is not part of this PR but you do not need the variable `@docker_commander`. You can just just the module name, the rspec helper `described_class` or define you subject.

Reply:
this should be fixed in the upcoming prs